### ISSUE TITLE
errors when BUILD_TESTING is enabled

### ISSUE THREAD

**Issue by anujm1** [openvkl/openvkl#12]
half.hpp should be updated to include this change:

https://sourceforge.net/p/half/code/415/

Otherwise, when -march is passed to gcc, it results in errors:

```
openvkl/0.13.0-r0/git/testing/volume/../external/half.hpp:911:62: error: '_MM_FROUND_TO_NEAREST_INT' was not declared in this scope
  911 |                                 (R==std::round_to_nearest) ? _MM_FROUND_TO_NEAREST_INT :
      |                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~
openvkl/0.13.0-r0/git/testing/volume/../external/half.hpp:912:63: error: '_MM_FROUND_TO_ZERO' was not declared in this scope
  912 |                                 (R==std::round_toward_zero) ? _MM_FROUND_TO_ZERO :
      |                                                               ^~~~~~~~~~~~~~~~~~
openvkl/0.13.0-r0/git/testing/volume/../external/half.hpp:913:67: error: '_MM_FROUND_TO_POS_INF' was not declared in this scope
  913 |                                 (R==std::round_toward_infinity) ? _MM_FROUND_TO_POS_INF :
      |                                                                   ^~~~~~~~~~~~~~~~~~~~~
openvkl/0.13.0-r0/git/testing/volume/../external/half.hpp:914:71: error: '_MM_FROUND_TO_NEG_INF' was not declared in this scope
  914 |                                 (R==std::round_toward_neg_infinity) ? _MM_FROUND_TO_NEG_INF :
      |                                                                       ^~~~~~~~~~~~~~~~~~~~~
openvkl/0.13.0-r0/git/testing/volume/../external/half.hpp:915:33: error: '_MM_FROUND_CUR_DIRECTION' was not declared in this scope
```

**Comment by gregjohnson** [comment]
Thanks for reporting this, looking into it.

**Comment by gregjohnson** [comment]
We've updated to half v2.2.0, which resolves this issue. This is available now in the `release-0.13.x` branch (https://github.com/openvkl/openvkl/commit/d111d9d8627bf1c13b9e4033a7982cf779d51954), and will also be in our next release.
